TestClient/MainWindow.cs

```diff
@@ -226,11 +228,12 @@ private void onStreamData(OBSWebsocket sender, StreamStatus data)
 
         private void btnConnect_Click(object sender, EventArgs e)
         {
-            if(!_obs.IsConnected)
+            if (!_obs.IsConnected)
             {
                 try
                 {
                     _obs.Connect(txtServerIP.Text, txtServerPassword.Text);
+                    folderpath.Text = _obs.GetRecordingFolder().ToString();
```

Comment:
Should move to OnConnect like the rest of the settings that are populated when connected

Reply:
ah nice, didn't realise it was there, thanks